clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -211,50 +230,13 @@ public FetchRequestData build() {
                     log.debug("Built full fetch {} for node {} with {}.",
                               nextMetadata, node, partitionsToLogString(next.keySet()));
```

Comment:
we log print sessionPartitions.keySet here since that's the full session data now.